### PR TITLE
Add vulnerability scan section to harbor docs

### PR DESCRIPTION
**Pull request by apdibbo**

### CODE REVIEW

File: source/ImproveTheDocs.rst [hunk 1]
```diff
@@ -11,3 +11,33 @@ If you have anything you would like to see documented then you can raise issues
 Alternatively, you can email cloud support and we can open an issue on your behalf.
 
 Any pull requests to improve or change the documentation, however minor, is welcome and greatly appreciated.
+
+Setting Up a local development environment
+------------------------------------------
+
+The following assumes you are using Windows Subsystem for Linux with Ubuntu 20.04 LTS with GitHub Desktop to manage interaction with GitHub.
```
Comment (DavidFair) [inline source/ImproveTheDocs.rst]: Could we tweak it to
`you are using Ubuntu 18.04 or 20.04 through Windows Subsystem for Linux or natively`

File: source/ImproveTheDocs.rst [hunk 2]
```diff
@@ -11,3 +11,33 @@ If you have anything you would like to see documented then you can raise issues
...
+
+  make html
+
+Assuming that this succeeds, you can then open ./build/html/index.html in your web browser
+
+Make any changes you wish to and then build again and test.
+
+A good resource for editing rst is here: https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst
+
+Once you are happy with your changes, commit the changes and publish and then create a pull request for us to review.
```
Comment (DavidFair) [inline source/ImproveTheDocs.rst:43]: Could we append `If your struggling feel free to open a PR with your changes so far and we'll be glad to provide assistance.`

File: source/Reference/Harbor.rst [hunk 3]
```diff
@@ -99,6 +99,21 @@ Some examples where a private projects should be considered are:
 
 Machines will not be able to pull from a private repository without first :ref:`logging in <login_harbor>`.
 
+.. _vuln_scan:
+
+Vulnerability scanner
+---------------------
+
+By default we configure all projects to automatically can images pushed to harbor for vulnerabilities and we also conduct weekly scans against all images.
```
Comment (DavidFair) [inline source/Reference/Harbor.rst]: Typo `can images pushed` -> `scan images pushed`

File: source/Reference/Harbor.rst [hunk 4]
```diff
@@ -99,6 +99,21 @@ Some examples where a private projects should be considered are:
...
+
+You can view the results by clicking into the project. Then into the repository and then you will see a summary of the vulnerabilities against each artifacts
+
+You can hover over the Vulnerabilities field to view a summary or click into the artifact to view further private_repo_details
```
Comment (DavidFair) [inline source/Reference/Harbor.rst]: `further private_repo_details` This feels like a typo?

File: source/Reference/Harbor.rst [hunk 5]
```diff
@@ -99,6 +99,21 @@ Some examples where a private projects should be considered are:
...
+
+We recommend that you resolve all Critical and High rated vulnerabilities as soon as possible.
+
+You (if you are an administrator of the project) can configure the project to disallow pulling of images based on the vulnerabilities against the image.
```
Comment (DavidFair) [inline source/Reference/Harbor.rst]: `You (if you are an administrator of the project)` -> `Project Administrators can configure`

**Review by DavidFair — CHANGES_REQUESTED** [review]
Some minor typos and clarifications, otherwise looks good

File: source/Reference/Harbor.rst (hunk 3 above)
Comment (apdibbo) [reply source/Reference/Harbor.rst]: Haha

**Comment by apdibbo** [comment]
I've addressed your comments

**Review by DavidFair — APPROVED** [review]
Thanks :+1:

**Pull request merged**